content/en/docs/operator.md

```diff
@@ -0,0 +1,61 @@
+
+
+## What is Operator Lifecycle Manager?
+
+Operator Lifecycle Manager (OLM) helps users install, update, and manage the lifecycle of all Operators and their associated services running across their clusters. It is part of the Operator Framework, an open source toolkit designed to manage Kubernetes native applications (Operators) in an effective, automated, and scalable way.
+
+
+## How does OLM enable cluster admins and Developers?
+
+The OLM is enabled by default in OpenShift Container Platform 4.X, which aids cluster administrators in installing, upgrading, and granting access to Operators running on their cluster. The OpenShift Container Platform web console provides management screens for cluster administrators to install Operators, as well as grant specific projects access to use the catalog of Operators available on the cluster.
+
+For developers, a self-service experience allows provisioning and configuring instances of databases, monitoring, and big data services without having to be subject matter experts, because the Operator has that knowledge baked into it.
+
+
+## Build with the Operator SDK
+
+The [Operator SDK](https://sdk.operatorframework.io/) provides the tools to build, test and package Operators. The Operator SDK strips away a lot of the boilerplate code that is normally required to integrate with the Kubernetes API. It also provides a usable scaffolding so developers can focus on adding business logic (for example, how to scale, upgrade, or backup the application it manages). Leading practices and code patterns shared across Operators are included in the Operator SDK to help prevent duplicating efforts. The Operator SDK also encourages short, iterative development and test cycles with tooling that allow for basic validation of the Operator, and automated packaging for deployment using the [Operator Lifecycle Manager](https://olm.operatorframework.io/).
+
+
+## Package with the Operator Lifecycle Manager
+
+With OLM, administrators can control which Operators are available in what namespaces and who can interact with running Operators. The permissions of an Operator are accurately configured automatically to follow a least-privilege approach. OLM manages the overall lifecycle of Operators and their resources, by doing things like resolving dependencies on other Operators, triggering updates to both an Operator and the application it manages, or granting a team access to an Operator for their slice of the cluster.
+
+Simple, stateless applications can use the Lifecycle Management features of the Operator Framework—without writing any code—by using a generic Operator (for example, the Helm Operator). However, complex and stateful applications are where an Operator can be especially useful. The managed-service capabilities that are encoded into the Operator code can provide an advanced user experience, automating such features as updates, backups and scaling.
+
```

Comment:
instead of using the links as `[Operator Framework](https://github.com/operator-framework)` the standard for docs is to create an alias and add the ref on the bottom of the page. e.g see the end oof the file; https://raw.githubusercontent.com/operator-framework/operator-sdk/master/website/content/en/docs/building-operators/golang/migration.md 

You will replace `[Operator SDK](https://sdk.operatorframework.io/)` with `[Operator SDK][operator-sdk]` and add at the end of the file `[operator-sdk]: https://sdk.operatorframework.io/`

Reply:
I have replaced all repetitive links in this format